src/components/SubmissionWizard/WizardComponents/WizardStepper.tsx

```diff
@@ -138,7 +138,7 @@ const WizardStepper = ({
   const params = useParams()
 
   const folderId = params.folderId
-  const newDraftPath = pathWithLocale(`newdraft/${folderId}`)
+  const newDraftPath = pathWithLocale(`submission/${folderId}`)
```

Comment:
I might be nitpicking but maybe the variable name could be something like `const submissionPath`?